proseco/gui.py

```diff
@@ -0,0 +1,375 @@
+# coding: utf-8
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import warnings
+import numpy as np
+from astropy.coordinates import SkyCoord, search_around_sky
+import astropy.units as u
+
+import agasc
+from Quaternion import Quat
+from Ska.quatutil import radec2yagzag
+import chandra_aca
+from mica.archive.aca_dark import get_dark_cal_image
+
+from .gui_char import CHAR, CCD
+
+# Ignore known numexpr.necompiler and table.conditions warning
+warnings.filterwarnings(
+    'ignore',
+    message="using `oa_ndim == 0` when `op_axes` is NULL is deprecated.*",
+    category=DeprecationWarning)
+
+
+STAR_CHAR = CHAR["Stars"]
+PIX_2_ARC = STAR_CHAR["General"]["Pix2Arc"]
+ARC_2_PIX = 1.0 / PIX_2_ARC
+RAD_2_PIX = 180/np.pi*3600*ARC_2_PIX
```

Comment:
Are you running PEP8 in your editor?

Reply:
Yes, and this doesn't actually flag in my editor.  I forget where this was copied from.